packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboardWithFilters.tsx

```diff
@@ -121,54 +121,40 @@ export const CloudPulseDashboardWithFilters = React.memo(
     });
 
     return (
-      <>
-        <Paper
-          sx={{
-            padding: 0,
-          }}
-        >
-          <Grid
-            justifyContent={{
-              sm: 'flex-end',
-              xs: 'center',
+      <Box display={'flex'} flexDirection={'column'} gap={2.5}>
```

Comment:
@nikhagra-akamai ,These much boxes we need?

Reply:
I've remove all the complex grid styling & replaced it with simple flex box